data/avro-schema/src/main/resources/avro/net/corda/data/membership/rpc/response/RegistrationStatusDetails.avsc

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
It would be good to move this to a common place also since it's used in `db.response.query` also